app.js

```diff
@@ -1,157 +1,209 @@
 'use strict';
 
 var correctAnswers = [];
-var wrongAnswers = [];
+// var wrongAnswers = [];
 
+//Wrapped as a function
+function greeting(){
 // Prompt the user to input their name.
 // Store user's name in a variable.
 // Ask the user eight questions and take their answer as input which will be stored in a variable.
 var userName = prompt('What\'s your name?');
 alert('Hello ' + userName + '. Thanks for coming to my site!');
 console.log('The user name is: ' + userName);
+return userName;
+}
+//Calling username function and assigning to userName variable
+var userName = greeting();
 
 // Question one.
-// Change user's input for the first question to all lower case.
-// Validate that their answer matches my answer or not.
-var answerOne = prompt('What city was I born in?').toLowerCase();
-console.log('The answer to question one is: ' + answerOne);
-// User's input for correct answer can be philadelphia or philly.
-// Depending on true or False, provide feedback.
-if(answerOne === philadelphia || answerOne === 'philly') {
-  alert('You got it right. I was born in Philly!');
-  correctAnswers.push(1);
-} else {
-  alert('Sorry, that\'s wrong. I was born in Philadelphia');
+//Wrapped question into a function
+function questionOne(){
+  // Change user's input for the first question to all lower case.
+  // Validate that their answer matches my answer or not.
+  var answerOne = prompt('What city was I born in?').toLowerCase();
```

Comment:
Good work modifying the variable input when the user gives it, rather than modifying it down below where we're checking it.